dbgpt/app/operators/rag.py

```diff
@@ -104,7 +104,7 @@ async def map(self, context: HOContextBody) -> List[Chunk]:
             return chunks
 
     metadata = ViewMetadata(
-        label=_("Knowledge Operator"),
+        label=_("Knowledge Space Operator"),
```

Comment:
The `label` and `description` comments are inconsistent. Maybe both of those are `Knowledge Space Operator`.